### PR TITLE
implemented mailto functionality

### PR DESCRIPTION
**Pull request by denis1stomin**
2 mailto-buttons using native email clients

### CODE REVIEW

File: webui/src/components/OKR.vue [hunk 1]
```diff
@@ -109,6 +115,24 @@
                 this.logChange(`Me deleted '${objId}'`);
             },
 
+            sendChangeSuggestion(objective) {
+                const targetSubject = this.$store.state.user.selectedSubject;
+                window.location = `mailto:${targetSubject.mail || targetSubject.userPrincipalName}?\
```
Comment (denis1stomin) [inline webui/src/components/OKR.vue]: why upn if no mail - https://support.cloudhq.net/it-admin-office-365-why-your-user-principal-name-upn-should-match-your-email-address/

File: webui/src/components/OKR.vue [hunk 2]
```diff
@@ -38,7 +38,7 @@
                 <div class="objective-icons">
                     <span @click="deleteObjective(objective.id)"><TrashIcon/></span>
                     <span><CopyIcon/></span>
-                    <span><SendIcon/></span>
+                    <span @click="sendChangeSuggestion(objective)"><SendIcon/></span>
```
Comment (denis1stomin) [inline webui/src/components/OKR.vue]: TODO : v-if="!canChangeOkr"

File: webui/src/components/OKR.vue [hunk 3]
```diff
@@ -109,6 +115,24 @@
                 this.logChange(`Me deleted '${objId}'`);
             },
 
+            sendChangeSuggestion(objective) {
+                const targetSubject = this.$store.state.user.selectedSubject;
+                window.location = `mailto:${targetSubject.mail || targetSubject.userPrincipalName}?\
+subject=Objective: ${objective.statement}&\
+body=Hi ${targetSubject.givenName || ''}.
```
Comment (denis1stomin) [inline webui/src/components/OKR.vue]: Cannot understand how to make body multi line.
Probably can do it later when will work on correct text.

File: webui/src/components/OKR.vue [hunk 4]
```diff
@@ -109,6 +109,24 @@
...
+OKR Portal ${window.location}`;
+            },
+
+            sendReminder() {
+                const targetSubject = this.$store.state.user.selectedSubject;
+                window.location = `mailto:${targetSubject.mail || targetSubject.userPrincipalName}?
+subject=Please fill objectives&
+body=Hi ${targetSubject.givenName || ''}%2C%0A
+Please fill objectives for the next period.%0A
+OKR Portal ${window.location}`;
```
Comment (preigile) [inline webui/src/components/OKR.vue]: please, add align

**Comment by preigile** [comment]
My be it will be better: "Please fill objectives for the next period on the OKR Portal http://localhost:1234/" or something like this. I mean one sente instead of two.

**Comment by preigile** [comment]
For `.objective-icons`, pls, set up `right: -85px;`

**Review by preigile — APPROVED** [review]

File: webui/src/components/OKR.vue (hunk 4 above)
Comment (denis1stomin) [reply webui/src/components/OKR.vue]: if you mean to add lines alignment, it will add extra white spaces in email content.

**Comment by denis1stomin** [comment]
> For .objective-icons, pls, set up right: -85px;

It will hurt delete button when mailto is visible. I suggest to postpone style in this case until new icons

**Pull request merged**